subworkflows/local/vcf_concatenate_germline/main.nf

```diff
@@ -1,42 +1,59 @@
-//
 // CONCATENATE Germline VCFs
-//
 
 // Concatenation of germline vcf-files
-include { ADD_INFO_TO_VCF                                     } from '../../../modules/local/add_info_to_vcf/main'
-include { TABIX_BGZIPTABIX as TABIX_EXT_VCF                   } from '../../../modules/nf-core/tabix/bgziptabix/main'
-include { BCFTOOLS_CONCAT  as GERMLINE_VCFS_CONCAT            } from '../../../modules/nf-core/bcftools/concat/main'
-include { BCFTOOLS_SORT    as GERMLINE_VCFS_CONCAT_SORT       } from '../../../modules/nf-core/bcftools/sort/main'
-include { TABIX_TABIX      as TABIX_GERMLINE_VCFS_CONCAT_SORT } from '../../../modules/nf-core/tabix/tabix/main'
+include { ADD_INFO_TO_VCF  } from '../../../modules/local/add_info_to_vcf/main'
+include { TABIX_BGZIPTABIX as TABIX_EXT_VCF } from '../../../modules/nf-core/tabix/bgziptabix/main'
+include { BCFTOOLS_NORM as GERMLINE_VCFS_NORM } from '../../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_CONCAT as GERMLINE_VCFS_CONCAT } from '../../../modules/nf-core/bcftools/concat/main'
+include { BCFTOOLS_SORT as GERMLINE_VCFS_CONCAT_SORT } from '../../../modules/nf-core/bcftools/sort/main'
+include { TABIX_TABIX as TABIX_GERMLINE_VCFS_CONCAT_SORT } from '../../../modules/nf-core/tabix/tabix/main'
 
 workflow CONCATENATE_GERMLINE_VCFS {
 
     take:
     vcfs
+    fasta
 
     main:
     versions = Channel.empty()
 
-    // Concatenate vcf-files
+    // Add additional information to VCF files
     ADD_INFO_TO_VCF(vcfs)
+    
+    // Compress the VCF files with bgzip
     TABIX_EXT_VCF(ADD_INFO_TO_VCF.out.vcf)
 
+    // Normalize the VCF files with BCFTOOLS_NORM
+    GERMLINE_VCFS_NORM(vcf: ADD_INFO_TO_VCF.out.vcf, fasta: fasta)
```

Comment:
I am surprised this runs. Nextflow does not have named inputs

Reply:
@FriederikeHanssen I don't think this ran. I fix it here https://github.com/eolaniru/sarek/tree/bcftools-norm_patch and alerted @JC-Delmas as I didn't want to raise a new PR

Reply:
no worries, I realised I got both PRs mixed up 🫠